odp_sdk/tabular_storage_client.py

```diff
@@ -0,0 +1,268 @@
+from typing import Dict, List, Optional
+from uuid import UUID
+
+import requests
+from pydantic import BaseModel, PrivateAttr
+
+from odp_sdk.dto import ResourceDto
+from odp_sdk.dto.table_spec import StageDataPoints, TableSpec
+from odp_sdk.dto.tabular_store import TableStage
+from odp_sdk.exc import OdpResourceExistsError, OdpResourceNotFoundError
+from odp_sdk.http_client import OdpHttpClient
+from odp_sdk.oqs.oqs_base import OqsBasePredicate
+
+
+class OdpTabularStorageController(BaseModel):
+    _http_client: OdpHttpClient = PrivateAttr(default_factory=OdpHttpClient)
+    tabular_storage_endpoint: str = "/data"
```

Comment:
The tabular storage endpoint should have a validator to ensure that it starts with a `/` and does not have one at the end